sdk/lib/opentelemetry/sdk/trace/samplers/constant_sampler.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
```

Comment:
Copyright year. JS recently removed the year (https://github.com/open-telemetry/opentelemetry-js/pull/1150). We can consider doing the same.

Reply:
I removed the year from the two new files. We can remove it from all other files in a separate PR.